python/packages/nisar/static/water_mask.py

```diff
@@ -0,0 +1,162 @@
+from __future__ import annotations
+
+import os
+
+import numpy as np
+from numpy.typing import ArrayLike
+from osgeo import gdal
+
+import isce3
+
+from .util import create_single_band_gtiff, make_scratch_file, unary_transform_blockwise
+
+
+def binarize_nisar_water_mask(water_distance: ArrayLike) -> np.ndarray:
+    r"""
+    Convert the input water distance map into a binary water mask.
+
+    Parameters
+    ----------
+    water_distance : array_like
+        The input water distance map, in the format specified by the NISAR Water Mask
+        Product Specification\ [1]_. A value of 0 indicates a water pixel. A value of
+        255 represents a no-data (invalid) pixel. Values in 1-200 represent non-water
+        pixels.
```

Comment:
```suggestion
        The input water distance map, in the format specified by the NISAR Water Mask
        Product Specification\ [1]_. A value of 0 indicates non-water pixel. A value of
        255 represents a no-data (invalid) pixel. Values in 1-200 represent water
        pixels.
```

Reply:
Note that this description corresponds to the input water distance map -- not the output binary water mask.

In the input water distance map, I expect that water pixels should have a value of 0 and non-water pixels will have a value of 1-200, depending on their distance from ocean or inland water.

So I think the current description is correct (but please correct me if I'm still misunderstanding!).